File: Java/Program/FasilitasUmum.java
//Class FasilitasUmum untuk simpan data tentang fasilitas
public class FasilitasUmum{
    private String kodeFasilitasUmum;
    private String namaFasilitasUmum;

    //Constructor kosong
    public FasilitasUmum(){
        this.kodeFasilitasUmum = "";
        this.namaFasilitasUmum = "";
    }

    //Constructor lengkap
    public FasilitasUmum(String kodeFasilitasUmum, String namaFasilitasUmum){
        this.kodeFasilitasUmum = kodeFasilitasUmum;
        this.namaFasilitasUmum = namaFasilitasUmum;
    }

    //Getter dan Setter untuk setiap atribut
    public String getKodeFasilitasUmum(){
        return kodeFasilitasUmum;
    }
    public void setKodeFasilitasUmum(String kodeFasilitasUmum){
        this.kodeFasilitasUmum = kodeFasilitasUmum;
    }

    public String getNamaFasilitasUmum(){
        return namaFasilitasUmum;
    }
    public void setNamaFasilitasUmum(String namaFasilitasUmum){
        this.namaFasilitasUmum = namaFasilitasUmum;
    }
}

File: CPP/Program/FasilitasUmum.cpp
#pragma once
#include <string>
using namespace std;

//Class FasilitasUmum untuk simpan data fasilitas
class FasilitasUmum{

    private:
    string kodeFasilitasUmum, namaFasilitasUmum;

    public:
    //Constructor kosong
    FasilitasUmum(){
        this->kodeFasilitasUmum = "";
        this->namaFasilitasUmum = "";
    }

    //Constructor dengan parameter untuk buat FasilitasUmum dengan data
    FasilitasUmum(string kodeFasilitasUmum, string namaFasilitasUmum){
        this->kodeFasilitasUmum = kodeFasilitasUmum;
        this->namaFasilitasUmum = namaFasilitasUmum;
    }

    //Getter & Setter untuk setiap atribut
    string getKodeFasilitasUmum(){
        return kodeFasilitasUmum;
    }
    void setKodeFasilitasUmum(string kodeFasilitasUmum){
        this->kodeFasilitasUmum = kodeFasilitasUmum;
    }

    string getNamaFasilitasUmum(){
        return namaFasilitasUmum;
    }
    void setNamaFasilitasUmum(string namaFasilitasUmum){
        this->namaFasilitasUmum = namaFasilitasUmum;
    }

    //Destruktor
    ~FasilitasUmum(){}
};